langs/pari-gp/Dockerfile

```diff
@@ -0,0 +1,59 @@
+FROM alpine:3.20 AS builder
+
+# install base deps
+RUN apk add --no-cache build-base curl gmp-dev readline-dev
+
+# download pari
+RUN curl https://pari.math.u-bordeaux.fr/pub/pari/unix/pari-2.15.5.tar.gz \
+  | tar xz
+
+# compile pari
+RUN mv pari-* pari       \
+ && cd pari              \
+ && ./Configure          \
+    --prefix=/usr        \
+    --mt=pthread         \
+ && make -j`nproc` gp    \
+ && make install         \
+ && make install-bin-sta \
+ # dunno what is
+    /usr/bin/gp
+
+# configure gp (doesnt cooperate, copying for now)
+#RUN touch ~/.gprc                           \
+#    histfile="/root/.gp_history" >> ~/.gprc \
+#    colors="darkbg"              >> ~/.gprc \
+#    lines=40                     >> ~/.gprc \
+#    parisizemax=4G               >> ~/.gprc \
+#    thraedsizemax=1G             >> ~/.gprc \
+#    read="/root/.gprc.gp"        >> ~/.gprc \
+# && touch ~/.gprc.gp
```

Comment:
Of course that doesn't cooperate, and why do I always catch it when it's out here.